apps/mobile/app/select-date/page.tsx

```diff
@@ -9,31 +9,32 @@ export default function page() {
   const MAX_LENGTH = 16;
 
   const [planName, setPlanName] = useState("");
-  const [isDisabledNextBtn, setIsDisabledNextBtn] = useState(true);
+  const [isDateSelected, setIsDateSelected] = useState(false);
+  const isRightName = planName.length > 0 && planName.length < 17;
+  const isActiveBtn = isRightName && isDateSelected;
 
   const handleChangeInput = (e: React.ChangeEvent<HTMLInputElement>) => {
     const { value } = e.currentTarget;
     setPlanName(value);
   };
 
-  const handleDisabledNextBtn = (isDisabled: boolean) => {
-    setIsDisabledNextBtn(isDisabled);
+  const handlSelecteDate = (isSelected: boolean) => {
```

Comment:
typo 잡아주면 좋을 것 같아요!!
```suggestion
  const handleSelecteDate = (isSelected: boolean) => {
```

Reply:
..... 헙 ..... 요러케 수정해둘게요 !!
```suggestion
  const handleSelectDate = (isSelected: boolean) => {
```